maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -27577,6 +27577,7 @@
 	icon_state = "0-2"
 	},
 /obj/machinery/firealarm/west,
+/obj/item/storage/box/psireceiver,
```

Comment:
this should be added in code to `/obj/structure/closet/secure_closet/RD` and not just placed in the map in strongdmm